internal/cmd/plugin/command_search.go

```diff
@@ -0,0 +1,98 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-yaml/yaml"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type Manifest struct {
+	Name         string `human:"Name" serialized:"name"`
+	Description  string `yaml:"description" human:"Description" serialized:"description"`
+	Dependencies string `yaml:"dependencies" human:"Dependencies" serialized:"dependencies"`
+}
+
+const cliPluginsUrl = "https://github.com/confluentinc/cli-plugins.git"
```

Comment:
Put this in the top-level command.go file since it's used in command_search.go and command_install.go?